ui/workItemGroup.html

```diff
@@ -0,0 +1,89 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Work item extension page sample</title>
+</head>
+
+<body style="overflow:auto;">
+    <script src="sdk/scripts/VSS.SDK.js"></script>
+
+    <script>
+
+        VSS.init({
+            explicitNotifyLoaded: true,
+            usePlatformScripts: true
+        });
+
+        VSS.require(["TFS/WorkItemTracking/Services"], function (_WorkItemServices) {
+
+            // Get the WorkItemFormService.  This service allows you to get/set fields/links on the 'active' work item (the work item
+            // that currently is displayed in the UI).
+            function getWorkItemFormService()
+            {
+                return _WorkItemServices.WorkItemFormService.getService();
+            }
+    
+            // Register a listener for the work item group contribution.
+            VSS.register("Fabrikam.samples-ui.sample-work-item-form-group", function () {
+                return {
+                    // Called when the active work item is modified
+                    onFieldChanged: function(args) {
```

Comment:
A group and page is also an observer.  There are no page/group specific methods.